config.yml

```diff
@@ -0,0 +1,46 @@
+defaults:
+  taskcluster:
+    rootUrl:        https://taskcluster-events-staging.herokuapp.com/ 
+    credentials:
+      clientId:     !env TASKCLUSTER_CLIENT_ID
+      accessToken:  !env TASKLCSUTER_ACCESS_TOKEN
+  
+  server:
+    port:           !env:number PORT
+    env:            !env NODE_ENV
+    forceSSL:       !env:flag FORCE_SSL
+    trustProxy:     false
+  
+  monitor:
+    mock:           true
+  
+  pulse:
+    username:       !env PULSE_USERNAME
+    password:       !env PULSE_PASSWORD
+    hostname:       !env PULSE_HOSTNAME
+    vhost:          /
```

Comment:
We probably want this to be configurable.

Reply:
Only in production, right?
And should we also have fakeAuth in tests?

Reply:
Configurable anywhere really. It will almost always be `/` but not always. We should use `fakeAuth`, but not important quite yet.

Reply:
Then should I make `hostname` configurable in `test` profile too?